web/tests/user_info_popovers.test.js

```diff
@@ -3,18 +3,13 @@
 const {strict: assert} = require("assert");
 
 const {$t} = require("./lib/i18n");
-const {mock_cjs, mock_esm, set_global, zrequire} = require("./lib/namespace");
+const {mock_esm, set_global, zrequire} = require("./lib/namespace");
 const {run_test} = require("./lib/test");
 const $ = require("./lib/zjquery");
 const {page_params} = require("./lib/zpage_params");
 
 const noop = function () {};
 
-class Clipboard {
```

Comment:
Now we can stop mocking the Clipboard, since it was in the `popovers.js` module and not in `user_info_popovers.js`.